orchestration/dagster_orchestration/hca_manage/manage.py

```diff
@@ -191,6 +200,32 @@ def submit_soft_delete(self, target_table: str, target_path: str) -> str:
 
         return response.json()["id"]
 
+    def submit_snapshot_request(self, optional_qualifier: Optional[str] = None) -> str:
+        date_stamp = str(datetime.today().date()).replace("-", "")
+        if optional_qualifier:
+            # prepend an underscore if this string is present
+            optional_qualifier = f"_{optional_qualifier}"
+        snapshot_name = f"{self.dataset}___{date_stamp}{optional_qualifier}"
+
+        reader_list = {
+            "dev": ["hca-snapshot-readers@dev.test.firecloud.org"],
+            "prod": ["hca-snapshot-readers@firecloud.org"]
+        }
+
+        response = self.data_repo_client.create_snapshot(
```

Comment:
⛏️ In fact, I think it'd be useful to output the entire payload we're submitting to Jade (i.e., so we can reconstruct a curl if something goes wrong).